src/api/status/public/index.html

```diff
@@ -15,10 +15,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <base href="/v1/status/" />
```

Comment:
You want to make sure that `charset` is defined first always, as there are some weird bugs where it has to happen in the first 1024 bytes.  I think you're fine here, but in general, do it first always.